packages/rum/src/apm-base.js

```diff
@@ -111,13 +111,15 @@ class ApmBase {
     } else {
       const loggingService = this.serviceFactory.getService('LoggingService')
       let message = "RUM Agent isn't correctly configured: "
+      const separator = ', '
 
       if (missing.length > 0) {
-        message += 'Missing config - ' + missing.join(', ')
+        message += 'Missing config - ' + missing.join(separator)
       }
 
       invalid.forEach(({ key, value, allowed }) => {
-        message += `, ${key} "${value}" contains invalid characters! (allowed: ${allowed})`
+        message += missing.length > 0 ? separator : ''
+        message += `${key} "${value}" contains invalid characters! (allowed: ${allowed})`
```

Comment:
I think it still doesn't work! for example if `missing.length==0` but there are multiple `invalid` errors. In this case they will be concatenated without any spacing.

Reply:
fixed.